types/react-i18next/index.d.ts

```diff
@@ -1,24 +1,40 @@
-// Type definitions for react-i18next 4.6
+// Type definitions for react-i18next 7.0
 // Project: https://github.com/i18next/react-i18next
 // Definitions by: Giedrius Grabauskas <https://github.com/GiedriusGrabauskas>
+//                 Simon Baumann <https://github.com/chnoch>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 2.6
```

Comment:
Is TypeScript 2.6 really the minimum required version? If you're not using any new features, best to keep this at 2.3 so that people don't have to be on 2.6 to use these typings.

Reply:
yup 2.3 should be sufficient

Reply:
Makes sense, have set it to 2.4, since the master version has changed in the meantime